mobile-app/app/screens/AppNavigator/screens/Auctions/components/BatchCard.tsx

```diff
@@ -112,10 +118,42 @@ export function BatchCard (props: BatchCardProps): JSX.Element {
             </View>
           </View>
           <View style={tailwind('flex flex-row items-center justify-center')}>
-            <CollateralTokenIconGroup
-              symbols={batch.collaterals.map(collateral => collateral.displaySymbol)}
-              maxIconToDisplay={3}
-            />
+            {batch?.highestBid?.owner !== address && batch.froms.some(bidder => bidder === address) &&
```

Comment:
Should we store this on a variable and maybe memoized it? So it doesn't recheck on every re-render